### ISSUE TITLE
[T7][Accessibility][Functional] –OBGYN button doesn’t read on hover

### ISSUE THREAD

**Issue by Channamma** [Mohanme13/Feb-19-Bug-bash#174]
OBGYN button doesn't read on hover
